### DISCUSSION TITLE
新版本v1.9.0的settings.py，如果redis有密码时，怎么写redis的配置

### DISCUSSION THREAD

**Discussion by czxin788** [hhyo/Archery#1746] [Q&A] [answered]
新版本v1.9.0的settings.py中，
默认redis配置如下，是无密码的配置：
CACHE_URL=(str, "redis://127.0.0.1:6379/0"),

如果redis有密码怎么写。

**Comment by peng19832** [comment]
![image](https://user-images.githubusercontent.com/9590209/184071446-0c3ee28b-5e11-4628-b8bd-c1dbb0d13afc.png)

**Reply by czxin788** [reply]
谢谢

**Comment by LeoQuote** [comment, marked as answer]
https://github.com/dpgmediamagazines/django-environ/blob/955aa7b84a793657d5ded2f3c08c0889c7e17ac6/environ/test.py#L468-L472
CACHE_URL 环境变量设置成 `redis://:redispass@127.0.0.1:6379/0` 就可以了

**Reply by czxin788** [reply]
亲测可用，感谢
